### PR TITLE
Build the Get Involved > Educate page

### PR DESCRIPTION
**Pull request by goplayoutside3**
## Package
lib-content
app-root

## Linked Issue and/or Talk Post
Builds on top of https://github.com/zooniverse/front-end-monorepo/pull/6205
Link to [Figma](https://www.figma.com/design/oZB2yOd3sjw3JTvUyz7lj1/Get-Involved-Pages?node-id=1-174&m=dev)

## Describe your changes

- Add content to the Educate component

## How to Review

- Run app-root locally and visit https://local.zooniverse.org:3000/get-involved/educate
- There's also a story component for Educate, but the images in the intro buttons will not show because the assets live in app-root's public folder
- Note that I did not include images in the Articles section because not all articles are from a Zooniverse-hosted website, so rather than risk copyright violations I left them as all non-image Article components

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General
- [ ] Tests are passing locally and on Github
- [ ] Documentation is up to date and changelog has been updated if appropriate
- [ ] You can `yarn panic && yarn bootstrap` or `docker-compose up --build` and FEM works as expected
- [ ] FEM works in all major desktop browsers: Firefox, Chrome, Edge, Safari (Use Browserstack account as needed)
- [ ] FEM works in a mobile browser

### CODE REVIEW

File: packages/lib-content/src/components/Article/Article.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ export default function Article({
         <Heading
           color={{ light: 'neutral-1', dark: 'accent-1' }}
           level={4}
-          size='1.5rem'
+          size={size !== 'small' ? '1.5rem' : '1.2rem'}
```
Comment (goplayoutside3) [inline packages/lib-content/src/components/Article/Article.js:54]: This change also affects the Community section on the signed-in and signed-out homepages, but overall improves the Article component's responsiveness.

**Comment by goplayoutside3** [comment]
In the interest of time, I’m going to self-review + merge the Get Involved content PRs today. These PRs for page content use pre-existing styling components from lib-content and static visuals. The PRs do not affect any public-facing production pages. I want to get the content up on fe-root.preview for re-routing planning and design review.

File: packages/lib-content/src/components/Article/Article.js (hunk 1 above)
Comment (seanmiller26) [reply packages/lib-content/src/components/Article/Article.js:54]: Smart choice here. I think it helps with the visual hierarchy once you're at that screen size.

**Comment by seanmiller26** [comment]
LGTM! 
A reminder for us to update the links under 'Resources for Educators - Group Stats, and Volunteer Certificate' once we have the various comms posted.

**Pull request merged**